HopsanGenerator/src/generators/HopsanFMIGenerator.cpp

```diff
@@ -1320,9 +338,9 @@ bool HopsanFMIGenerator::generateToFmu(QString savePath, ComponentSystem *pSyste
     //------------------------------------------------------------------//
 
     //Clean up temporary files
-   // cleanUp(savePath, QStringList() << "compile.bat" << modelName+".c" << modelName+".dll" << modelName+".so" << modelName+".o" << modelName+".hmf" <<
-   //         "fmiModelFunctions.h" << "fmiModelTypes.h" << "fmuTemplate.c" << "fmuTemplate.h" << "HopsanFMU.cpp" << "HopsanFMU.h" << "model.hpp" <<
-   //         "modelDescription.xml", QStringList() << "componentLibraries" << "fmu" << "HopsanCore");
+    cleanUp(savePath, QStringList() << "compile.bat" << modelName+".c" << modelName+".dll" << modelName+".so" << modelName+".o" << modelName+".hmf" <<
```

Comment:
No, several are deprecated. This function does nothing at all currently, since it tries to clean up in the savePath folder instead of the fmu-build folder. If we really wanted to clean up we should remove both fmu-build and fmu-stage, but I'm not sure we want to. Those files can be useful for debugging.